services/harmony/app/frontends/deployment-callback.ts

```diff
@@ -0,0 +1,37 @@
+import { Response } from 'express';
+import HarmonyRequest from '../models/harmony-request';
+import * as services from '../models/services/index';
+import env from '../util/env';
+import _ from 'lodash';
+
+/**
+ * Express.js handler that handls deployment callback message for deploying a service.
+ * This function will only be called when the service deployment is successful.
+ * Update the harmony server configuration for the new service image and service queue urls in the request.
+ * @param req - The request sent by the client
+ * @param res - The response to send to the client
+ */
+export default async function handleCallbackMessage(req: HarmonyRequest, res: Response): Promise<void> {
+  const logger = req.context.logger.child({ component: 'snsHandler.handleCallbackMessage' });
+  const headerSecret = req.headers['cookie-secret'];
+  const secret = process.env.COOKIE_SECRET;
+  logger.info(`handleCallbackMessage: ${JSON.stringify(req.body)}`);
+
+  if (headerSecret === secret) {
+    const { deployService, image, serviceQueueUrls } = req.body;
```

Comment:
I think it would be good to document what fields are in the message here. Not like JSON schema or anything, but maybe just a comment or define an interface above that documents the fields and use that here. Whichever, as long as there is some place that documents what the message fields are.